packages/compass-assistant/src/compass-assistant-provider.tsx

```diff
@@ -98,6 +108,7 @@ export const AssistantProvider: React.FunctionComponent<
 > = ({ chat, children }) => {
   const { openDrawer } = useDrawerActions();
   const track = useTelemetry();
+  const atlasAiService = useAtlasAiServiceContext();
```

Comment:
We should avoid exporting those from packages directly, instead you can wrap this provider in `createServiceProvider` and this will allow you to use locator inside this provider. The reason for this is to be very clear about the places where we use our dependency injection mechanism for other services